pkgs/development/compilers/ghc/8.2.2.nix

```diff
@@ -206,6 +206,8 @@ stdenv.mkDerivation (rec {
     "--disable-large-address-space"
   ];
 
+  dontAddExtraLibs = true;
```

Comment:
Could you please add a comment in front of those attributes that explains what they accomplish and/or why they are needed?

Reply:
Yes - done in latest in push.